### DISCUSSION TITLE
SSH Authentication

### DISCUSSION THREAD

**Discussion by SsNiPeR1** [community/community#27694] [Other Feature Feedback, Questions, & Ideas]
I have created a repo, added my key to it, then deleted this repo, created a new repo with the same name, then I am trying to add this key and github says that this key is already in use. Can the issue be solved _without_ creating a new keypair?

**Comment by airtower-luna** [comment]
I've seen mentions of that issue with deploy keys before, as far as I know there's no fix yet. You can try to [file a bug report with GitHub support](https://support.github.com/contact/bug-report).
